web/src/App.js

```diff
@@ -511,6 +512,7 @@ class App extends Component {
           <Route exact path="/payments" render={(props) => this.renderSigninIfNotSignedIn(<PaymentPage account={this.state.account} {...props} />)} />
           <Route exact path="/contact" render={(props) => this.renderSigninIfNotSignedIn(<ContactPage account={this.state.account} {...props} />)} />
           <Route path="/competitions" render={(props) => <CompetitionListPage account={this.state.account} {...props} />} />
+          <Route path="/conference" render={(props) => <Conference account={this.state.account} {...props} />} />
```

Comment:
It can be another conference like:

```
<Route exact path="/conferences/:conferenceName" render={(props) => this.renderSigninIfNotSignedIn(<ConferenceEditPage account={this.state.account} {...props} />)} />
```